From 8ae99ee447cf5f0160ea4ae978cdf37f5dcecd1e Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 21 Jul 2021 08:36:20 +0000
Subject: [PATCH] IWYU: missing memory include for unique_ptr usage in
 class_property.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix GCC build breakage because of missing inclide:
./../ui/base/class_property.h:120:58: error: ‘std::unique_ptr’ has not been declared
  120 |   T* SetProperty(const ClassProperty<T*>* property, std::unique_ptr<T> value);
      |                                                          ^~~~~~~~~~

Bug: 819294
Change-Id: I46b921876702b8d44674689bbb5acdc107db21e5
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3041030
Reviewed-by: Peter Boström <pbos@chromium.org>
Commit-Queue: José Dapena Paz <jdapena@igalia.com>
Cr-Commit-Position: refs/heads/master@{#903820}
---
 ui/base/class_property.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/base/class_property.h b/ui/base/class_property.h
index f7b2f559858b7..88b49386ae0ad 100644
--- a/ui/base/class_property.h
+++ b/ui/base/class_property.h
@@ -8,6 +8,7 @@
 #include <stdint.h>
 
 #include <map>
+#include <memory>
 #include <set>
 #include <type_traits>
 
